models/library-imported-from-ecore.json

```diff
@@ -73,7 +51,7 @@
     },
     "references": {
       "LIonCore_M3_Property_type": [
-        "INhBvWyXvxwNsePuX0rdNGB_J9hi85cTb1Q0APXCyJ0"
+        "LIonCore_stdlib_String"
```

Comment:
stdlib makes me think that there are many functionalities provided by this library. What about "common types" or "builtins"?

Reply:
You already use "builtins" in the Java impl., right? (Or "builtIns"?)

Reply:
Yes, I currently called it "builtins" but I can definitely change that

Reply:
Name changed.